### ISSUE TITLE
west espressif install throws error

### ISSUE THREAD

**Issue by szczys** [golioth/docs#56]
Following the quickstart, the `west espressif install` doesn't work for me: https://github.com/golioth/docs/blob/42a27c38152832125544e2e46f9a07a1e01fc3ae/docs/partials/install-espressif-toolchain.md#L4

```
$ west espressif install
usage: west [-h] [-z ZEPHYR_BASE] [-v] [-V] <command> ...
west: error: argument <command>: invalid choice: 'espressif' (choose from 'init', 'update', 'list', 'manifest', 'diff', 'status', 'forall', 'help', 'config', 'topdir', 'selfupdate')
```

Should this be `west espressif update` which I found [in the Zephyr docs](https://www.zephyrproject.org/zephyr-rtos-on-esp32/)?

**Comment by 0Grit** [comment]
west update brings in dependencies etc. In this case including the custom 'esspressif' command for west.

If the docs don't communicate this then this issue should in fact remain open.

I'm working on our developer onboarding flow this upcoming week. Feel free to open issues or ask questions https://github.com/golioth/docs/discussions

**Comment by szczys** [comment]
Fixed with #165
